vla/benchmarks/block_type_classification/train.py

```diff
@@ -208,7 +208,7 @@ def benchmark(model, preprocessor, train_json="train_los_dataset.json", test_jso
     else:
         feat_dim = extract_features_size(model, train_loader, device)
 
-    classifier = SimpleClassifier(feat_dim, num_classes, TrainConfig.output_activation).to(device)
+    classifier = SimpleClassifier(feat_dim, num_classes, TrainConfig.output_activation, num_hidden_layers=2).to(device)
```

Comment:
It might be convenient to introduce the number of hidden layers in the training config.